docs/universal-skills/changelog-generator/SKILL.md

```
@@ -0,0 +1,207 @@
+## Related Use Cases
+
+- Creating GitHub release notes
+- Writing app store update descriptions
+- Generating email updates for users
+- Creating social media announcement posts
+
+## WITH MCP Tools
+
+If you have MCP tools available, the process is faster:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)